test/clt-tests/tables-interaction/test-tables-interaction.rec

```diff
@@ -0,0 +1,91 @@
+––– block: ../base/dind/init –––
+––– input –––
+mkdir /tmp/local-index
+––– output –––
+––– input –––
+mkdir /tmp/dist-index
+––– output –––
+––– input –––
+docker pull -q --platform linux/x86_64 manticoresearch/manticore:latest > /dev/null; echo $?
+––– output –––
+0
+––– input –––
+docker pull -q --platform linux/x86_64 manticoresearch/manticore:6.2.12 > /dev/null; echo $?
```

Comment:
We can't hardcode 6.2.12 here. We need to use the latest release (which is tag `latest`)

Reply:
Done